presto-hive/src/test/java/com/facebook/presto/hive/TestHiveUtil.java

```diff
@@ -116,6 +142,39 @@ private static void assertToPartitionValues(String partitionName)
         assertEquals(actual, expected);
     }
 
+    private static void assertToPartitionNamesAndValues(String partitionName, List<String> expectedKeyList)
+            throws MetaException
+    {
+        Map<String,String> actual = toPartitionNamesAndValues(partitionName);
+        AbstractList<String> expectedValueList = new ArrayList<>();
+        for( String s: expectedKeyList) {
+            expectedValueList.add(null);
+        }
+        Warehouse.makeValsFromName(partitionName, expectedValueList);
+
+        if(actual.keySet().size() != expectedKeyList.size()){
+            assertTrue(false);
```

Comment:
replace this with 
`checkstate(actual.keySet().size() == expectedKeyList.size())`

Similarly at other place.